workflows/templates/cilium.deploy.yaml

```diff
@@ -105,4 +105,32 @@ spec:
                 value: |
                   echo "Waiting for cilium to start up"
                   cilium status --wait
-      
+        - name: "stash-network-policies"
+          dependencies:
+            - wait-for-cilium-status
+          templateRef:
+            name: ssh-template
+            template: shell-script
+          arguments:
+            parameters:
+              - name: dryRun
+                value: "{{inputs.parameters.dryRun}}"
+              - name: scriptContent
+                value: |
+                  kubectl get netpol -A
+                  rm -f /tmp/netpol.yaml
+                  for i in $(kubectl get netpol -A --no-headers | awk '{print $1}' | sort | uniq);do
```

Comment:
You can use `sort -u` instead of `sort | uniq`

Reply:
Changed